ee/fe/modules/340-monitoring-applications/monitoring/grafana-dashboards/applications/elasticsearch/elasticsearch.json

```
@@ -1936,91 +7224,113 @@
               "min": null,
               "show": false
             }
-          ]
+          ],
+          "yaxis": {
+            "align": false,
+            "alignLevel": null
+          }
         }
       ],
       "repeat": null,
-      "title": "Breakers",
+      "title": "Indices: Version map",
       "type": "row"
     }
   ],
-  "refresh": "30s",
-  "schemaVersion": 16,
+  "refresh": false,
+  "schemaVersion": 32,
   "style": "dark",
-  "tags": [],
+  "tags": [
+    "elasticsearch"
+  ],
   "templating": {
     "list": [
       {
         "current": {
+          "selected": true,
           "text": "default",
           "value": "default"
         },
+        "description": null,
+        "error": null,
         "hide": 0,
+        "includeAll": false,
         "label": "Prometheus",
+        "multi": false,
         "name": "ds_prometheus",
         "options": [],
         "query": "prometheus",
+        "queryValue": "",
         "refresh": 1,
         "regex": "",
+        "skipUrlSync": false,
         "type": "datasource"
       },
       {
         "allValue": null,
         "current": {},
-        "datasource": "$ds_prometheus",
+        "datasource": "${ds_prometheus}",
+        "definition": "",
+        "description": null,
+        "error": null,
         "hide": 0,
         "includeAll": false,
-        "label": "Namespace",
+        "label": "Сluster",
```

```suggestion
        "label": "Cluster",
```
Replace a Cyrillic letter with a Latin one.